pkg/apis/jaegertracing/v1/options.go

```diff
@@ -69,18 +86,24 @@ func (o Options) MarshalJSON() ([]byte, error) {
 }
 
 func (o *Options) parse(entries map[string]interface{}) {
-	o.opts = make(map[string]string)
+	o.opts = make(map[string]interface{})
```

Comment:
Don't we have a structure with this signature already?

Reply:
I can't find it, We have `FreeForm` but that is a different structure with` json *[]byte`